### DISCUSSION TITLE
laf 支持 Java 的讨论：设计、计划、痛点、方案、实践

### DISCUSSION THREAD

**Discussion by maslow** [labring/laf#163] [Generals]
laf 当前只支持 Node.js 运行时，需要支持更多语言的运行时。

我先提几个关键点做引子：

1. WebIDE 的支持
2. laf-cli 对 java 本地开发的支持
3. Java 开发者是否能接受 MongoDb 做为数据库，或者会需要进一步支持 MySQL / PG
4. 运行时以 Spring Boot 为底座是否过重，如果以 Vert.x / Netty 提供更轻的底座是否能匹配现有生态的开发者
5. Java 是编译型方案：基于 Maven 的依赖安装（动态 or 静态）、动态（热更新）执行函数等问题的方案

**Comment by SheepYang1993** [comment]
比较希望能加快对MySql的支持

**Reply by skyoct** [reply]
请问你对mysql应用场景是什么样子呢

**Reply by skyoct** [reply]
就在云函数内使用吗
